### DISCUSSION TITLE
Server-side login

### DISCUSSION THREAD

**Discussion by iebb** [iebb/F1WebViewer-SelfHosted#5] [Announcements]
This version allows providing credentials server-side, which is easier for keyboard-less clients. 
Don't use it in a public-available service, as it would still expose your token (though not username and password), which is dangerous.

<hr /><em>This discussion was created from the release <a href='https://github.com/iebb/F1WebViewer-SelfHosted/releases/tag/v0.2.0'>Server-side login</a>.</em>

**Comment by NewSingleHostReverseProxy** [comment]
Hello Sir,
I enjoy your great work :) can you please explain how I can config "providing credentials server-side"? :)
